content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -2,6 +2,282 @@
+   :alt: Disallowed expenses categories
+
+The :guilabel:`vehicle split` option available in the :guilabel:`Disallowed Expenses Report` allows
+you to see the rate and disallowed amount for each car.
+
+.. image:: belgium/vehicle-split.png
+   :alt: Disallowed expenses categories
+
+Fee form 281.50 and form 325
+============================
+
+Fee form 281.50
+---------------
+
+Annually, a **281.50 fee form** must be reported to the fiscal authorities. To do so, the tag
+`281.50` must be added on the **contact form** of the entities concerned by the **281.50** fee. To
+add the tag, go to the **Contacts** app, select the person or company you want to create a **281.50
+fee form** for, and add the `281.50` tag in the :guilabel:`Tags` field.
+
+.. image:: belgium/281-50.png
+   :alt: add the tag 281.50 on a contact form
+
+.. note::
+   Make sure the **street, zip code, country**, and **VAT number** are also informed on the
+   **Contact form**.
+
+Then, depending on the nature of the expense, add the corresponding `281.50` tag on the impact
+accounts. To do so, go to :menuselection:`Accounting --> Configuration --> Accounting: Chart of
+Accounts`, and click on :guilabel:`Setup` to add the corresponding `281.50` tag on the impacted
+accounts, ie.: :guilabel:`281.50 - Commissions`, depending on the nature of the expense.
+
+Form 325
+--------
+
+You can create a **325 form** by going to :menuselection:`Accounting --> Reporting --> Belgium:
+Create 325 form`. A new page pops up: select the right options and click :guilabel:`Generate 325
+form`. To open an already generated **325 form**, go to :menuselection:`Accounting --> Reporting -->
+Belgium: Open 325 forms`.
+
+.. image:: belgium/325-form.png
+   :alt: Add the tag 281-50 on a contact form
+
+CODA and SODA statements
+========================
+
+CODA
+----
+
+**CODA** is an electronic XML format used to import Belgian bank statements. You can download CODA
+files from your bank and import them directly into Odoo by clicking :guilabel:`Import Statement`
+from your :guilabel:`Bank` journal on your dashboard.
+
+.. image:: belgium/coda-import.png
+   :alt: Import CODA files
+
+.. note::
+   The :guilabel:`Belgium - Import Bank CODA Statements` module is installed by default when the
+   :guilabel:`Belgium - Accounting` and the :guilabel:`Belgium - Accounting Reports` modules are
+   installed.
+
+.. seealso::
+   :ref:`Import bank statements files <transactions/import>`
+
+SODA
+----
+
+**SODA** is an electronic XML format used to import accounting entries related to salaries. SODA
+files can be imported in the journal you use to record salaries by going to your Accounting
+**dashboard** and clicking :guilabel:`Upload` in the related journal card form.
+
+Once your **SODA** files are imported, the entries are created automatically in your 'salary'
+journal.
+
+.. image:: belgium/soda-import.png
+   :alt: Import SODA files
+
+Electronic invoicing
+====================
```

Comment:
This point also changes in a later version (I don't remember which) as the edi formats are then defined on the contact form and not in the journal anymore.